src/ontology/caro.Makefile

```diff
@@ -0,0 +1,8 @@
+## Customize Makefile settings for caro
+## 
+## If you need to customize your Makefile, make
+## changes here rather than in the main Makefile
+
+BFO_CLASSES_ONLY_URL = "https://raw.githubusercontent.com/BFO-ontology/BFO/master/bfo_classes_only.owl"
```

Comment:
such a weird component.. Should check with @dosumis if this is really needed. Sounds like an import to me, not a component.

Reply:
Yeah this was one i was not sure about - historic version had this file so I set it to download this instead of making it an import - I figured it was safer to have everything in then only selected things, esp since that file isn't too big. Happy to set it as import instead of component if @dosumis thinks its appropriate

Reply:
I am 90% sure this should be an import, not a component. Anything that is a component will be merged into the base file - why would we want BFO class hierarchies in the CARO base?

Reply:
switched to import